server/app/views/applicant/ProgramCardsSectionParamsFactory.java

```diff
@@ -262,6 +263,8 @@ public abstract static class ProgramCardParams {
 
     public abstract ImmutableList<String> categories();
 
+    public abstract long programId();
```

Comment:
Adding this because I need a stable and straightforward way to check for the program id of a card in browser tests. Without it, it was having to jump through too many hoops to get what I needed.